packages/fluentui/react-northstar/src/components/Tree/hooks/utils.ts

```diff
@@ -0,0 +1,9 @@
+import * as _ from 'lodash';
+
+export const removeItemAtIndex = (items: string[], itemIndex: number): string[] => {
+  return _.concat(_.slice(items, 0, itemIndex), _.slice(items, itemIndex + 1));
+};
```

Comment:
This function is really problematic, you tend to use it in iterations, and every time you call it you will trigger more iterations.... better to follow a pattern of identifying items to exclude and recreating the original array without them... you sacrifice more space for time

Reply:
Shift raise the similar concern. Going to change this to _.without :)